tests/test_sessions.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+import os
+import slack_today_i_did.reports as reports
+
+MOCK_PERSON = 'dave'
+MOCK_CHANNEL = '#durp'
+MOCK_TEXT = 'abcdflk'
+MOCK_TEST_FILE = '.testdata_sessions'
+
+
+def test_start_session():
```

Comment:
:art: `test_session_starting_and_adding`? since we're also testing `add_message`